src/main/java/io/confluent/connect/elasticsearch/bulk/BulkProcessor.java

```diff
@@ -623,6 +627,7 @@ public String toString() {
    * (mapper_parser_exception, illegal_argument_exception, and action_request_validation_exception)
    * resulting from bad records using the AK 2.6 reporter DLQ interface.
    */
+  @SuppressWarnings("serial")
```

Comment:
For my information, why did we choose to suppress the stacktrace here, instead of wrap the exception into a new exception, or even report the exception itself?

Reply:
https://github.com/confluentinc/kafka-connect-elasticsearch/pull/421#discussion_r444408464